contrib/epee/src/mlog.cpp

```diff
@@ -204,20 +204,22 @@ void mlog_configure(const std::string &filename_base, bool console, const std::s
           static_assert(std::is_integral<time_t>(), "bad time_t");
           return ta < tb;
         });
-        for (size_t i = 0; i <= found_files.size() - max_log_files; ++i)
+        const int max_index_found_files = static_cast<int>(found_files.size()) - static_cast<int>(max_log_files); // Prevents unsigned var overflowing
+        for (int i = 0; i < max_index_found_files; ++i)
         {
           try
           {
+            const boost::filesystem::path & found_file = found_files.at(i);
             boost::system::error_code ec;
-            boost::filesystem::remove(found_files[i], ec);
```

Comment:
@moneromooo-monero : Here's the potential UB. Replaced [] access with .at().

Reply:
I don't see it and I'm getting really suspicious it's cargo cult given the accompanying changes. Can you give details of how you end up with an out of bounds here ?